.github/workflows/lint.yml

```diff
@@ -1,12 +1,11 @@
-name: ci
+name: lint
 
 on:
-  pull_request:
   push:
-    branches: [ main ]
+  pull_request: [main, develop]
 
 jobs:
-  ci:
+  lint:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, '[skip-ci]')"
```

Comment:
To follow consistency should this be `'[skip-lint]'` instead of `'[skip-ci]'`?

Reply:
We could, but in the future if we add additional CI ideally this switch should also shut those down so this seems like the most generic and future-proof name.

Reply:
Right, but this ensures format and such. Not anything relevant to CI.

Reply:
Sure we could split up linting from other forms of CI and have different badges, I'll change it